### DISCUSSION TITLE
Support in Theia for Bndtools?

### DISCUSSION THREAD

**Discussion by scottslewis** [eclipse-theia/theia#13422] [Q&A]
I've done some cursory searches in the Theia issues/discussions for anything in Theia wrt supporting bndtools.   I've not found anything so far.   Is there any work that exists for using Theia to integrate and configure bndtools (and bndtools extension mechanisms...like repositories, workspace and project templates, third-party bndtools addons/plugins, etc).  Thanks, Scott

**Comment by msujew** [comment]
Hey @scottslewis,

I don't think anyone has worked on something like that. The most likely option is that someone has created a vscode extension that can be used in Theia, but a cursory google search only yielded [this thread](https://bnd.discourse.group/t/bndtools-for-vscode/255), which seems to indicate that there's nothing out there.
